### PR TITLE
Configure e2e tests

### PR DESCRIPTION
**Pull request by davvvid1**

### CODE REVIEW

File: e2e/pages/backendUrl.page.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+function fragments(text) {
+  const mapping = {
+    urlInput: element.bind(null, by.css('app-backend-url input')),
+    saveButton: element.bind(null, by.css('app-backend-url .save-button'))
+  };
+  return _.get(mapping, text);
+}
+
+function enterMockBackendUrl() {
+  return fragments('urlInput')().sendKeys('http://localhost:8060');
```
Comment (blabno) [inline e2e/pages/backendUrl.page.js]: Url should come as a parameter to `enterMockBackendUrl` function.

File: e2e/pages/backendUrl.page.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+function confirm() {
+  return fragments('saveButton')().click();
```
Comment (blabno) [inline e2e/pages/backendUrl.page.js]: We don't have to use `fragments` and string path. Simply reference `mapping.saveButton().sendKeys(...)`

File: e2e/pages/currency.page.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+const rp = require('request-promise');
+
+function fragments(text) {
+  const mapping = {
+    accounts: element.all.bind(null, by.css('app-account-manager .list .item'))
+  };
+  return _.get(mapping, text);
+}
+
+function seedAccounts() {
```
Comment (blabno) [inline e2e/pages/currency.page.js]: Seeding backend should be kept in dedicated service.

File: e2e/server.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+
+const port = 8060;
+const app = express();
+
+const db = {
```
Comment (blabno) [inline e2e/server.js]:
```
{
 get: {
  '/api/v1/payment-accounts': {payload:{}, status:200}
 },
 post: {
  '/api/v1/payment-accounts': {status:204}
 }
}
```

File: e2e/server.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+app.use(bodyParser.json());
+app.use(cors());
+
+app.post('/api/setup/payment-accounts', (req, res) => {
+  db.paymentAccounts.data.paymentAccounts = req.body.data;
+  db.paymentAccounts.status = req.body.status;
+  res.sendStatus(200);
+});
+
+app.get('/api/v1/payment-accounts', (req, res) =>
```
Comment (blabno) [inline e2e/server.js]: This can be done with a generic endpoint.

File: e2e/server.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+const app = express();
+
+const db = {
+  paymentAccounts: { data: { paymentAccounts: [] }, status: 200 }
+};
+
+app.use(bodyParser.json());
+app.use(cors());
+
+app.post('/api/setup/payment-accounts', (req, res) => {
```
Comment (blabno) [inline e2e/server.js]: Make it generic `app.post('/stub'`.
Payload:
```
{ payload, status, url}
```

**Review by blabno — CHANGES_REQUESTED** [review]

File: e2e/server.js (hunk 6 above)
Comment (davvvid1) [reply e2e/server.js]: Additionally, I added `method` property to determine `GET`, `POST`...

**Review by blabno — APPROVED** [review]

**Pull request merged**